src/config.rs

```diff
@@ -384,6 +387,11 @@ pub enum Config {
 
     /// Enable sending and executing (applying) sync messages. Sending requires `BccSelf` to be set
     /// and `Bot` unset.
+    ///
+    /// On real devices, this is usually always enabled and `BccSelf` is the only setting
+    /// that controls whether sync messages are sent.
+    ///
+    /// In tests, this is usually disabled.
```

Comment:
It's interesting that "usually" we don't need to test what is already tested by real devices. Maybe it's fine, just wanted to say how the comment reads as a whole

Reply:
I agree that it's not nice that tests test a different configuration than real devices, but it doesn't have anything to do with the current PR, this PR only documents the current behavior.